libs/insight-viewer/src/utils/HeatmapViewer/getRGBArray.ts

```diff
@@ -1,26 +1,29 @@
-/* eslint-disable no-bitwise */
-
 /**
  * A function that converts probability values
  * (0~1) of a specific area into [r, g, b] arrays corresponding to JET colormap colors
  */
 export function getRGBArray(value: number): number[] {
-  let r = 1.0
-  let g = 1.0
-  let b = 1.0
   const v = Math.max(Math.min(value, 1), 0)
+  let r, g, b
+
   if (v < 0.25) {
-    r = 0
-    g = 4 * v
+    // Between 0 and 0.25: 74,230,255 fixed
+    r = 74
+    g = 230
+    b = 255
   } else if (v < 0.5) {
-    r = 0
-    b = 1 + 4 * (0.25 - v)
-  } else if (v < 0.75) {
-    r = 4 * (v - 0.5)
-    b = 0
+    // Between 0.25 and 0.5: linearly convert from 74,230,255 to 221,255,0
```

Comment:
괄호를 넣어줘야 rbg임을 쉽게 알 수 있을 것 같습니다
```suggestion
    // Between 0.25 and 0.5: linearly convert from (74,230,255) to (221,255,0)
```

Reply:
말씀해주신 것처럼 괄호를 추가했습니다.  1160997